registrations/management/commands/transfer_registration_facility.py

```diff
@@ -0,0 +1,58 @@
+from django.core.management.base import BaseCommand, CommandError
+
+from hellomama_registration import utils
+from registrations.models import Registration
+
+
+class Command(BaseCommand):
+    help = ("Transfer all registrations linked to one facility to another "
+            "facility.")
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "--from-code", dest="from_code", type=str,
+            help=("The facility code where the registrations should be moved "
+                  "from.")
+        )
+        parser.add_argument(
+            "--to-code", dest="to_code", type=str,
+            help=("The facility code where the registrations should be moved "
+                  "to.")
+        )
+
+    def handle(self, *args, **kwargs):
+        from_code = kwargs['from_code']
+        to_code = kwargs['to_code']
+
+        if not from_code or not to_code:
+            raise CommandError('From and To code is required.')
+
+        for identity in utils.search_identities("details__personnel_code",
+                                                from_code):
+            from_identity = identity['id']
+            break
+
+        for identity in utils.search_identities("details__personnel_code",
+                                                to_code):
+            to_identity = identity['id']
+            break
```

Comment:
I think we'll have an issue here if we don't find any identities with that personnel code?

Maybe it's better if we check for `next(..)`, and catch the `StopIteration` and raise a `CommandError` instead?

Reply:
Also, do we have any guarantee that there will only be one identity for each personnel code? Maybe to just be safe, we should iterate through all of the `from_identity`s that we find, and do the below step for each identity, to make sure that we catch all of the registrations for that code?